client/question/services/FeedbackGeneratorService.js

```diff
@@ -402,50 +423,36 @@ tie.factory('FeedbackGeneratorService', [
        */
       getRuntimeErrorFeedback: function(feedbackDetails, rawCodeLineIndexes) {
         var errorInput = feedbackDetails.getErrorInput();
+        var errorLineNumber = feedbackDetails.getErrorLineNumber();
         var errorString = feedbackDetails.getErrorString();
         var language = feedbackDetails.getLanguage();
         var languageUnfamiliarityFeedbackIsNeeded = (
-          feedbackDetails.isLanguageUnfamiliarityFeedbackNeeded());
-
+            feedbackDetails.isLanguageUnfamiliarityFeedbackNeeded());
         var feedback = FeedbackObjectFactory.create(
-          FEEDBACK_CATEGORIES.RUNTIME_ERROR);
-        var fixedErrorString = errorString.replace(
-          new RegExp('line ([0-9]+)$'), function(_, humanReadableLineNumber) {
-            var preprocessedCodeLineIndex = (
-              Number(humanReadableLineNumber) - 1);
-            if (preprocessedCodeLineIndex < 0 ||
-                preprocessedCodeLineIndex >= rawCodeLineIndexes.length) {
-              throw Error(
-                'Line number index out of range: ' + preprocessedCodeLineIndex);
-            }
-            if (rawCodeLineIndexes[preprocessedCodeLineIndex] === null) {
```

Comment:
I think you still need this logic. It helps catch cases where the error arises in the preprocessed code.

Reply:
FYI @rabidbit -- this comment was not responded to.

Consider looking at the comment history or "Files Changed" in the future, in order to make sure no existing comments are outstanding, before requesting review.

Reply:
I did look at "Files Changed" (I actually did a search on 'seanlip' to make sure that I didn't miss any comments). But thanks for pointing these out, in case some of your comments are not displayed for whatever reason.